{{cookiecutter.project_slug}}/.flake8

```diff
@@ -12,3 +12,4 @@ ignore =
     E501,
     F841,   # Line too long
     E741,   # Ambiguous variable
+    W042,   # DEFAULT_AUTO_FIELD
```

Comment:
We should not encourage ignoring this warning. It's trying to get project to set an explicit value. Projects should do that, not ignore the warning. Particualrly new projects.

Reply:
You are right, I'm going to set the DEFAULT_AUTO_FIELD in `base.py`